config-importer/src/main/scala/Importer.scala

```diff
@@ -0,0 +1,358 @@
+package eoi
+package importer
+
+import java.time.LocalDate
+import spire.math.Interval
+import cats.implicits._
+import java.io._
+import scala.collection.immutable.ListMap
+import BigDecimal.RoundingMode.HALF_UP
+import Configurable._
+
+object Importer {
+
+  type YearAndCatMap = Map[Interval[LocalDate], Map[Char, Map[String, String]]]
+
+  // val autoAddBlankBands = List(
```

Comment:
this could be cleaned up too

Reply:
Actually this whole file is now unused (text-based conversion turned out to be so messy I had to go and do this using the ConfigWriter's in Importer2.scala). 
My plan is to remove the config-importer subproject entirely once we're happy with the config.